### ISSUE TITLE
Hard-coded PPP year in compute_spl() function's body

### ISSUE THREAD

**Issue by tonyfujs** [PIP-Technical-Team/wbpip#219]
Filing this so we don't lose track of this. I'm still not super comfortable with having the PPP years hard coded in the function's body... I don't have a solution for this, but I don't think we should have to update the function body of a `wbpip` function the day a new PPP release is available...

cc @randrescastaneda

**Comment by randrescastaneda** [comment]
You're right. And this is a problem that we have across PIP in general. There are many hard-coded values here and there. I think we should create a "meta" repo where all this information is hosted and updated. All the packages feed from that repo and that would be the only thing that we will need to update. It would play a similar role as the PFW for microdata. yet, it would make the process a little slows, since packages will have to constantly read from this "metadata" repo. I don't have clarity on what it would look like. 

What do you think?
